<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>119</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anna's Phone Booth</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_import_csv">
      <property name="text">
       <string>Import CSV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_show_sms">
      <property name="text">
       <string>Show received SMS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_sent_sms">
      <property name="text">
       <string>Show sent SMS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_call">
      <property name="text">
       <string>Call</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_send_sms">
      <property name="text">
       <string>Send SMS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>119</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
